### PR TITLE
update to use new cytometry_processing schema

### PR DESCRIPTION
**Pull request by lakikowolfe**

### CODE REVIEW

**Comment by ehfhcrc** [comment]
@lakikowolfe - there were a few changes in the previous commit in dev that need to be included to pass bioc-release branch (internal datasets needed to be updated).  Can you merge the updated `dev` back into `fb_cyto` and then push that up to make sure it passes?

**Comment by juyeongkim** [comment]
## Checklist for Reviewer

* [x]  Build it yourself and test the new features or changes
* [x]  Is it functional, and does it meet the requirements?
* [x]  Is it readable?
  * Easy to understand
* [x]  Is it following the coding standards/guidelines?
* [x]  Functional coding guide
* [x]  Style guide (No changes in `styler::style_pkg()`)
* [x]  Is it well documented (clear and sufficient)?
  * Comments make sense
  * Updated or new vignette or manual
* [x]  Does it have new/updated unit tests?
  * Use the `covr` package to examine the extent of test coverage
* [x]  Does it have new dependencies? Is it justifiable?
* [x]  Is there code duplication that should be reduced?
* [x]  Are there user interface improvements that could be made?
* [x]  Are there performance improvements that could be made?
* [ ]  Is it passing `R CMD check`?
* [ ]  Is it passing `BiocCheck::BiocCheck()`?

## Reviewer Comments
* Listing gating sets via `con$listGatingSet()` works in both all-study and study-level connections.
* However, summarizing and loading gating set do not work since the column name for "workspace ID" changed from `wsID` to `wsId` in your new `SelectedData` query.

```R
> library(ImmuneSpaceR)
> con <- CreateConnection("")

> con$summarizeGatingSet("SDY112_15-2011-V1_GS22")
Error in `[.data.table`(gsList, gatingSet == gating_set, wsID) : 
  j (the 2nd argument inside [...]) is a single symbol but column name 'wsID' is not found. Perhaps you intended DT[, ..wsID]. This difference to data.frame is deliberate and explained in FAQ 1.1.

> con$loadGatingSet("SDY112_15-2011-V1_GS22")
Error in `[.data.table`(gsList, gatingSet == gating_set, wsID) : 
  j (the 2nd argument inside [...]) is a single symbol but column name 'wsID' is not found. Perhaps you intended DT[, ..wsID]. This difference to data.frame is deliberate and explained in FAQ 1.1.
```

You can fix this issue by changing `wsID` to `wsId` in these lines of code:

https://github.com/RGLab/ImmuneSpaceR/blob/e4413c3853115472ec31a68a7b18a29fc14c002e/R/ISCon-cytometry.R#L121
https://github.com/RGLab/ImmuneSpaceR/blob/e4413c3853115472ec31a68a7b18a29fc14c002e/R/ISCon-cytometry.R#L157

* Lastly, as @ehfhcrc mentioned, there were changes in `dev` (new expected data) since you branched out from `dev`, and that's why some test cases fail. They are not related to your changes, so you don't have to merge the `dev` branch. I will just approve this PR once you fix the `wsId` issue.

**Comment by juyeongkim** [comment]
@lakikowolfe omg. I just noticed that it's `wsid` instead of `wsId`. Sorry about that.

File: R/ISCon-cytometry.R [hunk 1]
```diff
@@ -125,7 +125,7 @@ ISCon$set(
 
     # load gating set (RDS only) and merge metadata
     gs <- readRDS(
-      paste0(.buildGSPath(study, wsId, gatingSet), "/", gatingSet, ".rds")
+      paste0(.buildGSPath(study, wsid, gatingSet), "/", gatingSet, ".rds")
```
Comment (juyeongkim) [inline R/ISCon-cytometry.R]: "wsId" in Line 121 (inside of data.table) should change to `wsid` (column name)` not the on in Line 128 (a variable from Line 121).

**Pull request merged**